gazebo/sensors/DepthCameraSensor.cc

```diff
@@ -124,6 +124,20 @@ void DepthCameraSensor::Init()
     gzerr << "No world name" << std::endl;
   }
 
+  if (this->dataPtr->depthCamera->GetOutputReflectance())
+  {
+    ignition::transport::AdvertiseMessageOptions opts;
+    opts.SetMsgsPerSec(50);
+
+    this->dataPtr->imageReflectancePub =
+        this->node->Advertise<msgs::ImageStamped>(
+            this->Topic() + "_reflectance", 50);
+
+    this->dataPtr->imageReflectancePubIgn =
```

Comment:
I think this ignition topic is never published

Reply:
I guess if we don't need the ignition publisher, then it's fine to remove. @ahcorde what do you think?